### DISCUSSION TITLE
Upgrading from 3.3.5 to 3.3.14

### DISCUSSION THREAD

**Discussion by DantePasquale50** [community/community#34391] [Enterprise]
Hi All, This upgrade is failing with the log entries below. I'm not sure how to proceed. I have this feeling since no one applied the interim versions that this is the cause. But can anyone confirm this before I go through the upgrade chain!

You must correct your GRUB install devices before proceeding:

  DEBIAN_FRONTEND=dialog dpkg --configure grub-pc
  dpkg --configure -a
dpkg: error processing package grub-pc (--configure):
 installed grub-pc package post-installation script subprocess returned error exit status 1
Setting up libxml2-dev:amd64 (2.9.4+dfsg1-7+deb10u4) ...
Setting up python-twisted-runner (18.9.0-3+deb10u1) ...
Setting up libcurl4-openssl-dev:amd64 (7.64.0-4+deb10u3) ...
Setting up git (3:1~enterprise-3-3-release-2.30.0-3523-ge10e36af9166) ...
Setting up zlib1g-dev:amd64 (1:1.2.11.dfsg-1+deb10u2) ...
Setting up open-vm-tools (2:10.3.10-1+deb10u3) ...
Setting up gnupg (2.2.12-1+deb10u2) ...
Setting up libisccfg163:amd64 (1:9.11.5.P4+dfsg-5.1+deb10u7) ...
Setting up libxslt1-dev:amd64 (1.1.32-2.2~deb10u2) ...
Setting up libbind9-161:amd64 (1:9.11.5.P4+dfsg-5.1+deb10u7) ...
Setting up libirs161:amd64 (1:9.11.5.P4+dfsg-5.1+deb10u7) ...
Setting up bind9-host (1:9.11.5.P4+dfsg-5.1+deb10u7) ...
Setting up dnsutils (1:9.11.5.P4+dfsg-5.1+deb10u7) ...
Processing triggers for install-info (6.5.0.dfsg.1-4+b1) ...
Processing triggers for mime-support (3.62) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for libc-bin (2.28-10+deb10u1) ...
Processing triggers for systemd (241-7~deb10u8) ...
Processing triggers for man-db (2.8.5-2) ...
Processing triggers for ca-certificates (20220324+github1) ...
Updating certificates in /etc/ssl/certs...
0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
Errors were encountered while processing:
 grub-pc
W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Sub-process /usr/bin/dpkg returned an error code (1)
Removing 'local diversion of /usr/sbin/invoke-rc.d to /usr/sbin/invoke-rc.d.distrib'

**Comment by airtower-luna** [comment]
What kind of update are you talking about? :thinking: I only see part of what looks like package configuration on a Debian-ish system. If it's about some software hosted on GitHub, you should open an issue *in that repository* so the developers see it, or use whatever other support channels are listed there. :slightly_smiling_face:
